lib/rules/no-extra-parens.js

```diff
@@ -431,6 +435,10 @@ module.exports = {
                     return;
                 }
 
+                if (node.body.type === "ConditionalExpression" && ENFORCE_FOR_ARROW_CONDITIONALS) {
+                    return;
+                }
```

Comment:
@not-an-aardvark Thanks for answer! I understood this, so I deleted my comment :sorry: